### PR TITLE
update documentation for MeasureImageOverlap

### PR DESCRIPTION
**Pull request by karhohs**

### CODE REVIEW

File: cellprofiler/modules/measureimageoverlap.py [hunk 1]
```diff
@@ -8,7 +8,7 @@
 the white portions of two black and white images
```
Comment (mcquin) [inline cellprofiler/modules/measureimageoverlap.py]: Use the new module name in the documentation.

File: cellprofiler/modules/measureimageoverlap.py [hunk 2]
```diff
@@ -237,23 +242,26 @@ def create_settings(self):
...
+
+-  *{DM_KMEANS}:* Select to pick representative points using a K-Means
+   clustering technique. The foregrounds of both images are combined and
+   representatives are picked that minimize the distance to the nearest
+   representative. The same representatives are then used for the test
+   and reference images.
+-  *{DM_SKEL}:* Select to skeletonize the image and pick points
+   eqidistant along the skeleton.
+
+|image0|  *{DM_KMEANS}* is a choice that’s generally applicable to all
```
Comment (bethac07) [inline cellprofiler/modules/measureimageoverlap.py]: The image and the text after it should be indented

File: cellprofiler/modules/measureimageoverlap.py [hunk 3]
```diff
@@ -1,14 +1,14 @@
 # coding=utf-8
 
 """
-Calculate Image Overlap
+MeasureImageOverlap
```
Comment (bethac07) [inline cellprofiler/modules/measureimageoverlap.py]: Please adjust the length of the equal signs to match the module name.

File: cellprofiler/modules/measureimageoverlap.py [hunk 4]
```diff
@@ -237,23 +242,26 @@ def create_settings(self):
...
+   representative. The same representatives are then used for the test
+   and reference images.
+-  *{DM_SKEL}:* Select to skeletonize the image and pick points
+   eqidistant along the skeleton.
+
+    |image0|  *{DM_KMEANS}* is a choice that’s generally applicable to all
+    images. *{DM_SKEL}* is best suited to long, skinny objects such as
+    worms or neurites.
+
+.. |image0| image:: {PROTIP_RECOMEND_ICON}
```
Comment (bethac07) [inline cellprofiler/modules/measureimageoverlap.py]: This isn't rendering on my machine; try 
```
.. |image0| image:: memory:thumb-up.png
```
instead

File: cellprofiler/modules/measureimageoverlap.py [hunk 5]
```diff
@@ -165,70 +165,75 @@ def create_settings(self):
         self.ground_truth = cellprofiler.setting.ImageNameSubscriber(
```
Comment (bethac07) [inline cellprofiler/modules/measureimageoverlap.py]: Please add a docstring to the self.obj_or_imag setting above this one.

**Pull request merged**